src/__tests__/exampleData.js

```diff
@@ -125,6 +132,24 @@ export const stream: Stream = makeStream({
   description: 'An example stream.',
 });
 
+export const makeSubscription = (args: { stream?: Stream } = {}): Subscription => {
+  const { stream: streamInner = stream } = args;
+  return deepFreeze({
+    ...streamInner,
+    color: '#123456',
+    in_home_view: true,
+    pin_to_top: false,
+    audible_notifications: false,
+    desktop_notifications: false,
+    email_address: '??? make this value representative before using in a test :)',
+    is_old_stream: true,
```

Comment:
I couldn't find the meaning of `is_old_stream` in the codebase - can you please tell me? Also, it doesn't appear to be used anywhere at all ( I am probably wrong but a search in VS Code for `is_old_stream` gives only declarations, no uses. )

Reply:
From a scan of the server code, it seems to mean "this stream is old enough that the `stream_weekly_traffic` number is meaningful", and concretely "this stream is at least 7 days old".

It's not a great name :wink: 

Won't matter to us unless/until we start doing something with `stream_weekly_traffic` too.